supplements/tomb-of-annihilation/items.xml

```diff
@@ -0,0 +1,183 @@
+                <tr><td>21-35</td><td>You grow larger as if affected by an enlarge/reduce spell, except the effect lasts for 1 hour.</td></tr>
+                <tr><td>36-50</td><td>You grow smaller as if affected by an enlarge/reduce spell, except the effect lasts for 1 hour.</td></tr>
+                <tr><td>51-70</td><td>You arrive at the destination wearing nothing but the amulet of the black skull. Everything else that you were wearing or carrying appears in a random unoccupied space within 100 feet of you.</td></tr>
+                <tr><td>71-95</td><td>You are paralyzed for l minute or until this effect is ended with a lesser restoration spell or similar magic.</td></tr>
+                <tr><td>96-00</td><td>You become petrified. Th is effect can be ended only with a greater restoration spell or similar magic.</td></tr>
+            </table>
+        </description>
+        <setters>
+            <set name="category">Wondrous Items</set>
+            <set name="weight">0 lb.</set>
+            <set name="type">Wondrous Item</set>
+            <set name="rarity">Very Rare</set>
+            <set name="attunement">true</set>
+        </setters>
+    </element>
+
+<!-- BOOKMARK -->
+    <element name="Bookmark" type="Magic Item" source="Tomb of Annihilation" id="ID_WOTC_TOA_MAGIC_ITEM_BOOKMARK">
+        <description>
+            <p>This +3 dagger belongs to Artus Cimber (see appendix D). While you have the dagger drawn, you can use a bonus action to activate one of the following properties:</p>
+            <p class="indent">Cause a blue gem set into the dagger's pommel to shed bright light in a 20-foot radius and dim light for an additional 20 feet, or make the gem go dark.</p>
+            <p class="indent">Turn the dagger into a compass that, while resting on your palm, points north. </p>
+            <p class="indent">Cast dimension door from the dagger. Once this property is used, it can't be used again until the next dawn .</p>
+            <p class="indent">Cast compulsion (save DC 15) from the dagger. The range of the spell increases to 90 feet, but it targets only spiders that are beasts. Once this property is used, it can't be used again until the next dawn.</p>
+        </description>
+        <setters>
+            <set name="category">Magic Weapons</set>
+            <set name="cost" currency="gp">0</set>
+            <set name="weight" lb="0">0 lb.</set>
+            <set name="type" addition="dagger">Weapon</set>
+            <set name="rarity">Legendary</set>
+            <set name="enhancement">3</set>
+            <set name="weapon">Dagger</set>
+            <set name="attunement">true</set>
+        </setters>
+        <rules>
+            <grant type="Spell" id="ID_PHB_SPELL_DIMENSION_DOOR" />
+            <grant type="Spell" id="ID_PHB_SPELL_COMPULSION" />
+        </rules>
```

Comment:
```suggestion

```
Our policy is to only grant spells if you are also able to cast the spell using a spell slot or if its a cantrip